content/rs/references/client_references/client_csharp.md

```diff
@@ -1,61 +1,210 @@
+In order to use Redis in .NET you need a .NET Redis client. In following sections, we demonstrate the use of [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis), a general purpose Redis client. Additional .NET clients for Redis can be found under the [C# section](https://redis.io/clients#c-sharp) of the Redis Clients page.
 
 ## Installing StackExchange.Redis
 
-StackExchange.Redis' installation instructions are given in the ["Installation" section](https://stackexchange.github.io/StackExchange.Redis/) of its documentation page. It can be installed via the nuget package manager console with the following command:
+There are several ways to install this package including:
 
-    PM> Install-Package StackExchange.Redis
+1. With the .NET CLI: 
+```sh
+dotnet add package StackExchange.Redis
+```
+2. with the package manager console: 
+```sh
+`PM> Install-Package StackExchange.Redis`
+```
+3. With the NuGet GUI in Visual Studio   
 
 ## Opening a Connection to Redis Using StackExchange.Redis
 
-The following code creates a connection to Redis using StackExchange.Redis:
+The following code creates a connection to Redis using StackExchange.Redis in the context of a trivial console application:
 
-    using StackExchange.Redis;
+```csharp
+using StackExchange.Redis;
+using System;
+using System.Threading.Tasks;
+
+namespace ReferenceConsoleRedisApp
+{
+    class Program
+    {
+        static readonly ConnectionMultiplexer redis = ConnectionMultiplexer.Connect(
+            new ConfigurationOptions{
+                EndPoints = {"localhost:6379"}                
+            });
+        static async Task Main(string[] args)
+        {
+            var db = redis.GetDatabase();
+            var pong = await db.PingAsync();
+            Console.WriteLine(pong);
+        }
+    }
+}
+
+```
+
+The above example assumes that you have a Redis Server running locally.
+
+To configure the connection to your environment, adjust the parameters in the [ConfigurationOptions](https://stackexchange.github.io/StackExchange.Redis/Configuration) object appropriately. For the remainder of the examples the configuration will simply be `localhost`.
+
+## Connection Pooling with StackExchange.Redis
+
+While StackExchange.Redis does not provide direct means for conventional connection pooling, we recommend you **share and reuse** the ConnectionMultiplexer object. The ConnectionMultiplexer object should not be created per operation - it is to be created only once at the beginning and reused for the duration of the run. ConnectionMultiplexer is thread-safe so it can be safely shared between threads. For more information, refer to StackExchange.Redis’s [Basic Usage document](https://stackexchange.github.io/StackExchange.Redis/Basics).
```

Comment:
```suggestion
StackExchange.Redis does not support conventional connection pooling.  We recommend **sharing and reusing** the ConnectionMultiplexer object. The ConnectionMultiplexer object should not be created for each operation.  Instead, create an instance at the beginning and then reuse the object throughout your process. ConnectionMultiplexer is thread-safe so it can be safely shared between threads. For more information, see the  [Basics document](https://stackexchange.github.io/StackExchange.Redis/Basics).
```

Reply:
Why is sharing and reusing bolded here?  It shouldn't be unless there's a semantic reason.  (It's not a term of art, so it doesn't need to be defined as one, nor does it refer to a UX element.)

Reply:
It was in there originally, I think I meant to underline the fact that that's really the behavior you should be using - I could  go either way on having it bolded - if the style guides specifically call for it not to be bolded that's a good enough reason to un-bold it.

Reply:
We use bolding and emphasis for specific situations.  I'd remove unless there's a good reason not to.

Reply:
done